### PR TITLE
Add HoloPlay to Media/Audio Streaming

### PR DESCRIPTION
**Pull request by stephane-r**
Previously closed because the first release was less than 4 months ago : 

https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/121

Now it's good 😄 

Sorry for new PR, i can't re-open the first PR posted in september.

---

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/holoplay.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - React
```
Comment (Rabenherz112) [inline software/holoplay.yml:9]:
```suggestion
  - Nodejs
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
Otherwise looks fine for me.

On a side note, though this is not directly related to this review, I noticed that you don't have a LICENSE file in your repository. While mentioning it in the readme is absolutely fine, maintaining a separate file for it is best practice. It also allows external programs, including GitHub, to identify the type of license you're using.

File: software/holoplay.yml (hunk 1 above)
Comment (stephane-r) [reply software/holoplay.yml:9]: Mmm, i'm not sur for this change. It's not an Node.js application. It's an client only app using React. The server on Dockerfile is an basic nginx image.

Of cours, it use Node.js, but only for compilation process, not on production environment.

Comment (ryangurn) [reply software/holoplay.yml:9]: Personally I would be okay with Node because react uses node.

**Review by ryangurn — APPROVED** [review]

File: software/holoplay.yml (hunk 1 above)
Comment (Rabenherz112) [reply software/holoplay.yml:9]: > It's an client only app using React. The server on Dockerfile is an basic nginx image.

I understand what you mean, but at least as far as I can see from your Getting Started section, there are two main deployment methods for the Software - utilizing npm in development mode (which implies Nodejs), or employing a docker image. Hence, for me, the principal server-side prerequisites are Nodejs or Docker, given that React is not necessary for the server-side deployment

Comment (stephane-r) [reply software/holoplay.yml:9]: I've no problem to add Node.js :+1: 

For the production server : you can use Ruby, PHP, or other server you like. The production build is an HTML, CSS and JS files :) Node.js is only required for the compilation process.

Comment (Ki-er) [reply software/holoplay.yml]: Just to add to this https://github.com/stephane-r/holoplay-pwa?tab=readme-ov-file#setup lists npm making it a requirement.

File: software/holoplay.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
```
Comment (Ki-er) [inline software/holoplay.yml:4]:
```suggestion
description: "Save music to favorites and create playlists (alternative to Youtube)."
```

Comment (Ki-er) [inline software/holoplay.yml:4]:
```suggestion
description: "Web app using Invidious API for listening to Youtube audio sources."
```

File: software/holoplay.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Nodejs
```
Comment (Ki-er) [inline software/holoplay.yml:9]: Changed order to match documentation.

```suggestion
platforms:
  - Nodejs
  - Docker
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
I have suggested two different description edits depending om which is the more popular. If the former is popular, I have suggested some grammar changes.

File: software/holoplay.yml (hunk 1 above)
Comment (stephane-r) [reply software/holoplay.yml]: Node.js and npm is require only for build step or development environment. You can use HoloPlay without build by using release source or Docker image.

I'll update the README for add this detail 👍

File: software/holoplay.yml (hunk 2 above)
Comment (nodiscc) [reply software/holoplay.yml]: I don't think `(alternative to Youtube)` is appropriate, it's just a web app that uses Invidious API to consume content from Youtube.

**Review by nodiscc — APPROVED** [review]
Thanks for the reviews

**Pull request merged**